pages/index.tsx

```diff
@@ -22,19 +163,83 @@ const Home: NextPage = () => {
       </Head>
 
       <main className={styles.main}>
-        <h1 className={styles.title}>
-          (First) AidKit Task
-        </h1>
-        <ul className={styles['applicant-list']}>
-          {applicants.map(a => <li className={styles.applicant} key={a.name}>
-            <div className={styles.name}>{a.name}</div>
-            <div className={styles.phone}>{a.phone}</div>
-          </li>
-          )}
-        </ul>
+        {loading ? (
+          <div>Loading...</div>
+        ) : (
+          <>
+            <h1 className={styles.title}>(First) AidKit Task</h1>
+            <input
+              placeholder="Search"
+              value={search}
+              onChange={(event) => handleChange(event.target.value)}
+            />
+            <div className={styles.filters}>
+              Filter by Screener:
+              {filterOptions.map((option) => (
+                <label key={option}>
+                  <input
+                    type="radio"
+                    name="filter"
+                    value={option === "all" ? "" : option}
+                    checked={filter === (option === "all" ? "" : option)}
+                    onChange={handleFilterChange}
+                  />
+                  {option.charAt(0).toUpperCase() + option.slice(1)}
+                </label>
+              ))}
+            </div>
+            <div className={styles["table-container"]}>
```

Comment:
To fix the jumpiness we talked about in the interview, the table just needed to be wrapped in a div with a fixed height. This keeps the items in the `<main>` flex container in place while allowing the table to grow and shrink as needed inside its own parent. 🤦